### PR TITLE
seam 0.0.39 (new formula)

### PR DESCRIPTION
**Pull request by mastafit**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/s/seam.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+  test do
+    system "#{bin}/seam", "config", "set", "fake-server"
+    output = pipe_output("#{bin}/seam health get_health")
+    assert_match "I’m one with the Force. The Force is with me.", output
+  end
+end
```
Comment (chenrui333) [inline Formula/s/seam.rb:21]:
```suggestion
  end

  test do
    system bin/"seam", "config", "set", "fake-server"
    output = shell_output("#{bin}/seam health get_health")
    assert_match "I’m one with the Force. The Force is with me.", output
  end
end
```

**Comment by SMillerDev** [comment]
@mastafit please do not create new pull requests for every change. It makes them very hard to review.

**Comment by SMillerDev** [comment]
> Sorry, I was wondering what is the correct way to do changes to my pr without failing Commit style test

You can squash them before pushing.

**Review by chenrui333 — DISMISSED** [review]

**Review by p-linnane — DISMISSED** [review]

**Comment by p-linnane** [comment]
@mastafit Please cherry-pick this into a new PR from a non-org fork so we can get it merged.

**Pull request closed without merging**